simulator/src/main/java/Simulator.java

```diff
@@ -20,22 +21,19 @@ public Simulator(Map<Arg, Object> args)
 
     public void simulate()
     {
-        ((VirtualDeviceWrapper)this.devices.toArray()[0]).phoneSendBluetoothPacket(new Packet(1,0, "Hello Device no.1, How are you?").getBytes());
-        sleep(5000);
-        ((VirtualDeviceWrapper)this.devices.toArray()[2]).phoneSendBluetoothPacket(new Packet(3,2, "Hello Device no.3, How was your day?").getBytes());
-        ((VirtualDeviceWrapper)this.devices.toArray()[1]).phoneSendBluetoothPacket(new Packet(0,1, "Hello Device no.0! I am Good. How are you?").getBytes());
+        // Initialize Devices with Bluetooth INIT PACKET
+        ((VirtualDeviceWrapper)this.devices.toArray()[0]).phoneSendBluetoothPacket(new Packet(0,1, 1).getBytes());
```

Comment:
Is it possible to make this a dynamic loop so we can simulator a variable number of devices?

Reply:
It is but once again i think this is a nice to have once we have the router working on the devices